packages/ui-react/src/pages/ScreenRouting/PlaylistScreenRouter.tsx

```diff
@@ -37,6 +36,12 @@ const PlaylistScreenRouter = () => {
     return <ErrorPage title={t('empty_shelves_heading')} message={t('empty_shelves_description')} />;
   }
 
+  // register playlist screens
+  playlistScreenMap.registerDefault(PlaylistGrid);
+  if (type === PLAYLIST_TYPE.playlist) {
+    playlistScreenMap.registerByContentType(PlaylistLiveChannels, PLAYLIST_CONTENT_TYPE.live);
+  }
```

Comment:
This feels a bit hacky, why is this needed?

Reply:
It will work then this way (out of component's scope):

```
// register playlist screens
playlistScreenMap.registerDefault(PlaylistGrid);
playlistScreenMap.register(PlaylistLiveChannels, (item) => item?.contentType === PLAYLIST_CONTENT_TYPE.live && item.type === PLAYLIST_TYPE.playlist);

```

Reply:
We don't need a separate router screen per se, but perhaps we can create a separate screen map for content? 

```ts
const screenMap = type === 'playlist' ? playlistScreenMap : contentScreenMap;
```

The `contentScreenMap` only contains the default screen using the PlaylistGrid.